torch/_utils.py

```diff
@@ -52,70 +52,49 @@ def _type(self, dtype=None, non_blocking=False, **kwargs):
     return dtype(self.size()).copy_(self, non_blocking)
 
 
-def _hpu(self, device=None, non_blocking=False, **kwargs):
-    """Returns a copy of this object in HPU memory.
+def _backend(self, backend_name, device=None, non_blocking=False, **kwargs):
+    """Returns a copy of this object in backend memory.
 
-    If this object is already in HPU memory and on the correct device, then
+    If this object is already in backend memory and on the correct device, then
     no copy is performed and the original object is returned.
 
     Args:
-        device (int): The destination HPU id. Defaults to the current device.
+        device (int): The destination device id. Defaults to the current device.
```

Comment:
I know this is existing code, but I think we should rename this to `device_idx` to make it clear that this is just the device index and not the full device including devicetype

Reply:
changed